docs/faq.md

```diff
@@ -114,3 +117,30 @@ LLDAP is hobbyist software developed in good will by volunteers. LLDAP is not su
 The project is not too complex and is even kept minimalist on purpose. However, unless you'd like to audit and maintain the codebase in the foreseeable future, it's not recommended to adopt LLDAP for the infrastructure of your big organization.
 
 You are free to use LLDAP for any purpose, as long as you respect the copyleft. However, please do not complain if feature X is not implemented, or if the volunteers are not fixing problems fast enough for your taste. LLDAP is a project born of love and adventure, not a commercial endeavour.
+
+## Does LLDAP need Internet access?
+
+By default, LLDAP uses CSS, fonts and JS served from third-party [Content Delivery Networks](https://en.wikipedia.org/wiki/Content_delivery_network). This has upsides and downsides, which are being evaluted in issue [#1219](https://github.com/lldap/lldap/issues/1219).
+
+It is possible to run LLDAP without Internet access, in a LAN setup, which is the default configuration for our provided Docker images. You need to make sure the `index.html` file served by `lldap` from the configured `assets_path` is actually the `app/index_local.html` file. Then, download the external resources to the same folder. Assuming you use the default `assets_path = "./app/"`:
+
+```sh
+for file in $(cat app/static/libraries.txt); do wget -P app/static "$file"; done
+for file in $(cat app/static/fonts/fonts.txt); do wget -P app/static/fonts "$file"; done
+```
+
+LLDAP should now be working on your LAN without requiring Internet access on the server or client side.
+
+## Why does my browser make requests to external HTTP services?
+
+By default, LLDAP uses CSS, fonts and JS served from third-party [Content Delivery Networks](https://en.wikipedia.org/wiki/Content_delivery_network). This has upsides and downsides, which are being evaluted in issue [#1219](https://github.com/lldap/lldap/issues/1219).
+
+If you are worried about the privacy and security of your setup, it is also possible to serve all static assets locally without 3rd parties. See the question [Does LLDAP need Internet access?](#does-lldap-need-internet-access) of this FAQ.
+
+## What is LLDAP's privacy policy?
```

Comment:
From the wikipedia definition: "A privacy policy is a statement or legal document (in privacy law) that discloses some or all of the ways a party gathers, uses, discloses, and manages a customer or client's data."

I would start with something to the effect of "LLDAP doesn't have a privacy policy because it doesn't collect any data."